patches/components-history-core-browser-sync-delete_directive_handler.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/history/core/browser/sync/delete_directive_handler.cc b/components/history/core/browser/sync/delete_directive_handler.cc
+index aed3aec1f04f15f9f806887110d87abac9ef06ba..13053b7fe8a0281e3f9d9f69b58bf35e5d199e2e 100644
+--- a/components/history/core/browser/sync/delete_directive_handler.cc
++++ b/components/history/core/browser/sync/delete_directive_handler.cc
+@@ -387,7 +387,6 @@ bool DeleteDirectiveHandler::CreateUrlDeleteDirective(const GURL& url) {
```

Comment:
chromium_src override the entire method and return false

Reply:
fixed at https://github.com/brave/brave-core/pull/20329/commits/fa0fe5fc915c0d66b13ff2e16a1c7e27c11118b5